extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -249,6 +249,7 @@ export class GoogleSubscriptionsPlatform {
         granted: true, //swgEntitlements.getEntitlementForThis makes sure this is true.
         grantReason: GrantReason.SUBSCRIBER, // there is no other case of subscription for SWG as of now.
         dataObject: swgEntitlement.json(),
+        decryptedDocumentKey: swgEntitlements.decryptedDocumentKey,
```

Comment:
This should certainly be tested. Alternatively, you can remove it for now and add later once swg-js library is upgraded.

Reply:
adding TODO